utils/utils.jsx

```diff
@@ -253,6 +266,10 @@ export function extractFirstLink(text) {
     // strip out inline markdown images
     inText = inText.replace(/!\[[^\]]*]\([^)]*\)/g, '');
 
+    // remove markdown *, ~~ and _ characters
+    inText = loopReplacePattern(inText, /(\*|~~)(.*?)\1/, '$2');
```

Comment:
Instead of the function, could this instead be done by using `inText = inText.replace(/.../g, '$2');`?

Reply:
Before we extracted it as a function, we have tried the `/g` flag. 
But it seemed that `/g` was not enough to deal with nested structures of Markdown.
Because the original JavaScript regex doesn't support balanced matching or recursive matching, we decided to manually wrote a loop function to do so.

Reply:
Cool. Makes sense